examples/example_plugin/example_plugin/ui/index.js

```diff
@@ -1,11 +1,16 @@
 export { default as ExampleAppFullWidthPageComponent } from "./FullWidthPage"
+export { default as ExamplePluginRetrieveViewOverride } from "./RetrieveView"
 
 const app_config = {
     detail_tabs: {},
     full_width_components: {
-        "dcim:sites": ["ExampleAppFullWidthPageComponent"]
+        "dcim:devices": ["ExampleAppFullWidthPageComponent"]
     },
-    view_overrides: {}
+    view_overrides: {
+        "dcim:platforms": {
+            "retrieve": "ExamplePluginRetrieveViewOverride"
```

Comment:
We can't have `example_plugin` override any core views as this will break tests for the core view (and rightly so). This is what the `example_plugin_with_view_override` plugin is explicitly for, and that's where this override should be implemented.